to.etc.domui/src/main/java/to/etc/domui/dom/html/Page.java

```diff
@@ -1380,4 +1390,31 @@ public void execute(@NonNull IExecute code) {
 				page.addDelayedExecution(code);
 		}
 	}
+
+	/**
+	 * Checks if page can be left.
+	 * @return
+	 */
+	public boolean canLeaveCurrentPage(boolean handleNavigationWhenModified) {
```

Comment:
This should not be called by any user code. Name it internalCanLeaveCurrentPage().

Reply:
named it internal